src/components/shared/modal/modalFadeBox/modalFadeBox.scss

```diff
@@ -0,0 +1,16 @@
+// Copyright (c) Microsoft. All rights reserved.
+
+@import 'src/styles/mixins';
+@import 'src/styles/themes';
+
+.modal-fade-box {
+  position: absolute;
+  width: 100%;
+  height: 100%;
+  opacity: 0.5;
+  z-index: 0;
+
+  @include themify($themes) {
+    background-color: themed('colorFlyoutDropShadow');
```

Comment:
See above for creating modal specific color names.

Reply:
done